### PR TITLE
Implement missing API calls

### PR DESCRIPTION
**Pull request by kesar**
Fixes https://github.com/skipperbent/tinder-php-sdk/issues/4

### CODE REVIEW

File: src/Pecee/Http/Service/Tinder.php [hunk 1]
```diff
@@ -57,6 +57,10 @@ protected function authenticate() {
         }
     }
 
+	public function meta() {
```
Comment (kesar) [inline src/Pecee/Http/Service/Tinder.php:60]: there is a new meta call

screen shot 2016-01-01 at 23 40 51

File: src/Pecee/Http/Service/Tinder.php [hunk 2]
```diff
@@ -77,12 +81,16 @@ public function like($userId) {
...
+		return $this->api('like/' . $userId .'/super/');
+	}
+
     public function pass($userId) {
         return $this->api('pass/' . $userId);
     }
 
-    public function updates() {
-        return $this->api('updates');
+    public function updates($lastActivityTime = '') {
```
Comment (kesar) [inline src/Pecee/Http/Service/Tinder.php:92]: updates now its a POST method that require last_activity_date. If last_activity_date its not passed its going to show all the messages and matches.

Comment (kesar) [inline src/Pecee/Http/Service/Tinder.php:92]: screen shot 2016-01-01 at 23 42 37

**Comment by skipperbent** [comment]
Great - nice work and thanks for helping this project staying up to date :+1:

**Pull request merged**